.github/workflows/cibuildwheel.yml

```diff
@@ -64,6 +64,9 @@ jobs:
         run: |
           python -m build --sdist
           python ci/export_sdist_name.py
+        env:
+          # Prevent including development runtime dependencies in metadata.
+          CIBUILDWHEEL: 1
```

Comment:
This can be dropped, as the Meson build doesn't use the environment like setuptools did.